packages/css/markprompt.css

```diff
@@ -0,0 +1,853 @@
+:root {
+  --markprompt-background: #ffffff;
+  --markprompt-foreground: #171717;
+  --markprompt-muted: #fafafa;
+  --markprompt-mutedForeground: #737373;
+  --markprompt-border: #e5e5e5;
+  --markprompt-input: #ffffff;
+  --markprompt-primary: #6366f1;
+  --markprompt-primaryForeground: #ffffff;
+  --markprompt-secondary: #fafafa;
+  --markprompt-secondaryForeground: #171717;
+  --markprompt-primaryHighlight: #ec4899;
+  --markprompt-secondaryHighlight: #a855f7;
+  --markprompt-overlay: #00000010;
+  --markprompt-ring: #0ea5e9;
+  --markprompt-radius: 8px;
+  --markprompt-text-size: 0.875rem;
+  --markprompt-button-icon-size: 1rem;
+}
+
+@media (prefers-color-scheme: dark) {
+  :root {
+    --markprompt-background: #050505;
+    --markprompt-foreground: #d4d4d4;
+    --markprompt-muted: #171717;
+    --markprompt-mutedForeground: #737373;
+    --markprompt-border: #262626;
+    --markprompt-input: #ffffff;
+    --markprompt-primary: #6366f1;
+    --markprompt-primaryForeground: #ffffff;
+    --markprompt-secondary: #0e0e0e;
+    --markprompt-secondaryForeground: #ffffff;
+    --markprompt-primaryHighlight: #ec4899;
+    --markprompt-secondaryHighlight: #a855f7;
+    --markprompt-overlay: #00000040;
+    --markprompt-ring: #ffffff;
+  }
+}
+
+:where([class^='Markprompt']) {
+  box-sizing: border-box;
+}
+
+:where(
+    [class^='Markprompt'] *,
+    [class^='Markprompt'] *:before,
+    [class^='Markprompt'] *:after
+  ) {
+  box-sizing: inherit;
+}
+
+:where(.MarkpromptTrigger, .MarkpromptClose) {
+  all: unset;
+}
+
+:where(.MarkpromptTrigger) {
```

Comment:
Is this not equivalent to `.MarkpromptTrigger`? (same for all below)

Reply:
No, it is less specific making it easier to override. Users can now target the classes with `.MarkpromptTrigger` and be sure that their styles always win, regardless of the DOM order of the styles. See https://polypane.app/css-specificity-calculator/#selector=%3Awhere(.MarkpromptTrigger) and https://polypane.app/css-specificity-calculator/#selector=.MarkpromptTrigger for specificity scores.

Reply:
Ok got it 👍

Reply:
Nice! TIL